### ISSUE TITLE
Create <Emoji /> component

### ISSUE THREAD

**Issue by zenorocha** [resendlabs/react-email#534]
Having a safe way to add emojis to your email would be a great addition to React Email.

> What is the recommended approach for working with emojis? Specifically, I want to insert Github emojis into my HTML (and later export these templates). Should I use SVGs or is there a better way?

Notes:
* Most email clients don't support SVG
* Should we render a PNG or wrap a HTML entity? http://www.amp-what.com/unicode/search/emoticon

Useful resources:
* https://dreamyguy.github.io/react-emojis
* https://github.com/twitter/twemoji

**Comment by hiteshmodi0624** [comment]
When working with emojis in email templates, it's important to keep in mind that not all email clients support all types of emojis. While some email clients may support SVGs, others may not, so it's generally a good idea to use a fallback method as well.

One common approach is to use PNGs to display the emojis. This can be done by downloading the PNGs from a library like Twemoji, which provides a set of emojis that can be used for free. Once you have the PNGs, you can include them in your email templates using standard HTML <img> tags.

Another approach is to use HTML entities to display the emojis. HTML entities are special codes that can be used to represent characters that can't be displayed directly in HTML. There are a number of HTML entities that can be used to display emojis, and you can find a list of them on sites like amp-what.com. Once you know the code for the emoji you want to use, you can include it in your HTML using the appropriate entity.

When it comes to React Email, you may want to consider using a library like react-emojis to simplify the process of adding emojis to your templates. This library provides a set of React components that can be used to display emojis using either PNGs or HTML entities, depending on the browser's capabilities.

In summary, the best approach for working with emojis in email templates will depend on your specific needs and the capabilities of your target email clients. Using PNGs or HTML entities are both valid options, and you may want to consider using a library like react-emojis to simplify the process.

**Comment by zenorocha** [comment]
Chat GPT detected 😅

**Comment by bdsqqq** [comment]
> - Should we render a PNG or wrap a HTML entity? http://www.amp-what.com/unicode/search/emoticon

I'd lean on wrapping an HTML entity. Doing PNGs implies choosing a specific emoji set that may or may not match the user's system. This is exaggerated by the fact that the body might include an emoji used in the headline where it would use the system's style.

On the other side, PNGs add big maintenance costs but allow the library to ship new emojis even before some operating systems. (eg: WhatsApp takes this approach so Android users don't have to deal with "���" emojis when chatting with iPhone users)

**Comment by bruno88cabral** [comment]
last month I created an example using emojis, I just copied and pasted from the original email
 https://demo.react.email/preview/codepen-challengers
